app/change_sets/hyrax/dynamic_change_set.rb

```diff
@@ -1,14 +1 @@
 # frozen_string_literal: true
```

Comment:
Is there a reason to leave this one line behind?

Reply:
Nope, it was just WIP when you reviewed it